plugins/tekton/src/components/PipelineRunList/PipelineRunRow.tsx

```diff
@@ -46,12 +58,28 @@ type PipelineRunRowProps = {
 type PipelineRunNameProps = { row: PipelineRunKind };
 
 const PipelineRunName = ({ row }: PipelineRunNameProps) => {
+  const classes = useStyles();
   const name = row.metadata?.name;
+  const signed =
+    row?.metadata?.annotations?.[chainsSignedAnnotation] === 'true';
 
   return (
     <div>
       {name ? (
-        <ResourceBadge color={tektonGroupColor} abbr="PLR" name={name || ''} />
+        <ResourceBadge
+          color={tektonGroupColor}
+          abbr="PLR"
+          name={name || ''}
+          suffix={
+            signed ? (
+              <Tooltip title="Signed">
```

Comment:
Action button uses PF tooltip, so can we use PF tooltip instead material ui here as well?
![image](https://github.com/janus-idp/backstage-plugins/assets/9964343/c64ddaef-ebde-4864-a385-9bf0159e6e49)